src/tt.h

```diff
@@ -43,14 +43,29 @@ struct TTEntry {
 
   void save(Key k, Value v, Bound b, Depth d, Move m, Value ev, uint8_t g) {
 
-    if (m || (k >> 48) != key16) // Preserve any existing move for the same position
-        move16 = (uint16_t)m;
-
-    key16     = (uint16_t)(k >> 48);
-    value16   = (int16_t)v;
-    eval16    = (int16_t)ev;
-    genBound8 = (uint8_t)(g | b);
-    depth8    = (int8_t)d;
+      if (key16 != (k >> 48))
+      {
+          key16     = (uint16_t)(k >> 48);
+          move16    = (uint16_t)m;
+          value16   = (int16_t)v;
+          eval16    = (int16_t)ev;
+          genBound8 = (uint8_t)(g | b);
+          depth8    = (int8_t)d;
+      }
+      else
+      {   
+          if (m) move16 = (uint16_t)m; // Preserve any existing move for the same position
```

Comment:
Can you use the use the standard indentation, please ?

    if (m) // Preserve any existing move for the same position
        move16 = (uint16_t)m;